### ISSUE TITLE
assertAlmostEqual conversion breaks on comments

### ISSUE THREAD

**Issue by Palpatineli** [dannysepler/pytestify#46]
environment:
windows 10 wsl2 Ubuntu 22.10
python 3.10.6
pytestify 1.41.

symptom:
When there is a comment at the end of the line of a `self.assertAlmostEqual`, pytestify correctly strips the closing parenthesis of the `self.assertAlmostEqual`, but then attempts to inject the closing parenthesis of `pytest.approx` at the end of the line, after the comment, creating a syntax error that breaks ast.py and skip the file.

some changes need to be done to pytestify/fixes/asserts.py line 261
however because # can be in strings or escaped, the proper fix may need ast level manipulation.
The conceptually easier solution may just be not to strip the original closing parenthesis.

**Comment by dannysepler** [comment]
aha, i see this on 

```python
self.assertAlmostEquals(a, b)  # hi
```

which gets rewritten to 

```python
assert a == pytest.approx(b# hi)
```

**Comment by dannysepler** [comment]
should now be fixed in v1.5.0 -- appreciate all the context you added in your description!
